Added DOCTYPE content in HTML (#443)
Codecademy/docs: content/html/concepts/tags/terms/doctype/doctype.md

---

> Title: '<doctype>' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed

Reviewer: Great job finding using the templates! Let's actually remove these comments so we are just left with the necessary text.

> Title: '<doctype>' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed

Reviewer: ```suggestion
Title: '<!DOCTYPE>'
```

It is valid to name this `<!doctype>` since this tag is not case-sensitive. However, let's keep this in caps per convention and we can make a **Note:** at the bottom of this entry if this is desired.

> Description: 'The <!DOCTYPE> tag tells the browser that the current file is in the HTML format.' # Required; ideally under 150 characters (used in search engine results and content previews)

Reviewer 2: ```suggestion
Description: 'Tells the browser that the current file is in the HTML format.' # Required; ideally under 150 characters (used in search engine results and content previews)
```

Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
  - 'Web Development'
  - 'Web Design'
Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!

>   - 'Browsers'

Reviewer 2: Let's use at least two (2) tags for each entry.

CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<!DOCTYPE>` tag tells the users browser that the current file is HTML and not another type such as JS or CSS

Reviewer: ```suggestion
The `<!DOCTYPE>` tag tells the browser that the current file is written in HTML and not another type such as JavaScript or CSS.
```

(Style point) HTML and CSS are good abbreviated, for JavaScript we'll write this out

> ## How to Use

Reviewer: While I appreciate the brevity/simplicity of this entry, I do think it can be enhanced by adding a "Syntax" heading above this with just a code snippet of the tag like so:

## Syntax

```html
<!DOCTYPE html>
```

The "How to Use" heading can be changed to "Example" (the example below is great as is).

> To use in a HTML file, simply add the `<!DOCTYPE>` tag to the top of your HTML

Reviewer: ```suggestion
The `<!DOCTYPE>` tag is added to the top of an HTML file.
```

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Title of the document</title>
  </head>

  <body>

>     The content of the document......

Reviewer: ```suggestion
    The content of the document...
```

---
Title: '<!DOCTYPE>'
Description: 'Tells the browser that the current file is in the HTML format.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Browsers'
  - 'HTML'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<!DOCTYPE>` tag tells the browser that the current file is written in HTML and not another type such as JavaScript or CSS.

Reviewer 2: ```suggestion
The `<!DOCTYPE>` tag tells the browser that a given file is a markup (`.html`) file and that the browser must apply full web standards to it.
```

What's written in an HTML file isn't strictly limited to pure HTML. You can use a `<script>` tag to write custom JavaScript and a `<style>` tag in the `<head>` to apply CSS styles.
